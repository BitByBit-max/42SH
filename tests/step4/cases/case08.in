#!/bin/bash

input_string="123456"

case $input_string in
    # match a string that is a number
    [[:digit:]]*)
        echo "The string is a number"
        ;;

    # match any other string
    *)
        echo "The string is not a number"
        ;;
esac
