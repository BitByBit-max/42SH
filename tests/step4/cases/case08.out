The string is a number
